CoreUtilities/src/au/gov/asd/tac/constellation/utilities/javafx/JavaFxUtilities.java

```diff
@@ -17,6 +17,7 @@
 
 import javafx.scene.paint.Color;
 
+// TODO: This whole class can be deleted when the histogram rewrite fully replaces the old histogram
```

Comment:
noting that this class won't be needed once everything is fully cut over, would it better to simply define the function within the Histogram View module (specifically the HistogramDisplay class where it is being used) rather than here? If it were intending to stay in the long term then this would be an appropriate place to put it, but since its a temporary solution, it may be best left where it is used, as it will be easier to remember to be removed afterwards